src/Symfony/Component/Console/Application.php

```diff
@@ -128,6 +130,19 @@ public function setSignalsToDispatchEvent(int ...$signalsToDispatchEvent): void
         $this->signalsToDispatchEvent = $signalsToDispatchEvent;
     }
 
+    public function setAlarmInterval(?int $interval): void
```

Comment:
this needs some phpdoc explaining what is the unit of that internal.
Alternatively, rename the parameter to `$seconds` (same name than in the SignalRegistry)

Reply:
Done, https://github.com/symfony/symfony/compare/791685d099b00f050785f3010ab289cf06567122..33fb7b7a503113689e9ce3474e21b9e120865d39